### DISCUSSION TITLE
汉字识别错了

### DISCUSSION THREAD

**Discussion by gongdj** [PaddlePaddle/PaddleOCR#13353] [Q&A]
使用paddlepaddle 3.0.0b0的版本，paddleocr2.8，识别模型为ch_PP-OCRv4_det_server_infer，汉字竣能识别对，换成：ch_PP-OCRv4_rec_infer.tar时，被识别成了：峻，有办法解决吗？
**换成ch_PP-OCRv4_det_server_infer准确率很高，但识别用时太多了**

ocr = PaddleOCR(use_angle_cls=True, lang="ch", det_db_unclip_ratio=2.0, use_gpu=False, det=False,
                        enable_mkldnn=True, cpu_threads=8,
                        det_model_dir="F:\\github\\ocr_model\\ch_PP-OCRv4_det_server_infer")
**根据上面的ocr，识别出错误的字：竣**

ocr = PaddleOCR(use_angle_cls=True, lang="ch", det_db_unclip_ratio=2.0, use_gpu=False, det=False,
                        enable_mkldnn=True, cpu_threads=8,
                        det_model_dir="F:\\github\\ocr_model\\ch_PP-OCRv4_det_server_infer",
                        rec_model_dir='F:\\github\ocr_model\\ch_PP-OCRv4_rec_server_infer')
**根据上面的ocr，识别出正确的字：峻**

依赖为：
(paddle3_env) PS F:\github\PaddleOCR> pip list
Package               Version
--------------------- -----------
anyio                 4.4.0
astor                 0.8.1
beautifulsoup4        4.12.3
blinker               1.8.2
certifi               2024.6.2
charset-normalizer    3.3.2
click                 8.1.7
colorama              0.4.6
contourpy             1.1.1
cycler                0.12.1
Cython                3.0.10
decorator             5.1.1
exceptiongroup        1.2.1
fire                  0.6.0
Flask                 3.0.3
fonttools             4.53.1
h11                   0.14.0
httpcore              1.0.5
httpx                 0.27.0
idna                  3.7
imageio               2.34.2
imgaug                0.4.0
importlib_metadata    8.0.0
importlib_resources   6.4.0
itsdangerous          2.2.0
Jinja2                3.1.4
kiwisolver            1.4.5
lazy_loader           0.4
lmdb                  1.5.1
lxml                  5.2.2
MarkupSafe            2.1.5
matplotlib            3.7.5
networkx              3.1
numpy                 1.24.4
opencv-contrib-python 4.10.0.84
opencv-python         4.10.0.84
opt-einsum            3.3.0
packaging             24.1
paddleocr             2.8.0
paddlepaddle          3.0.0b0
pillow                10.3.0
pip                   24.0
protobuf              5.27.1
pyclipper             1.3.0.post5
pyparsing             3.1.2
python-dateutil       2.9.0.post0
python-docx           1.1.2
PyWavelets            1.4.1
PyYAML                6.0.1
rapidfuzz             3.9.4
requests              2.32.3
scikit-image          0.21.0
scipy                 1.10.1
setuptools            69.5.1
shapely               2.0.4
six                   1.16.0
sniffio               1.3.1
soupsieve             2.5
termcolor             2.4.0
tifffile              2023.7.10
tqdm                  4.66.4
typing_extensions     4.12.2
urllib3               2.2.2
Werkzeug              3.0.3
wheel                 0.43.0
zipp                  3.19.2

**Comment by UserWangZz** [comment]
单纯的模型预测错误，server模型大小比infer模型要大很多，性能也会比infer模型高

**Reply by gongdj** [reply]
但也存在部分字，server和infer都识别不了，中文，有办法解决吗？

**Reply by UserWangZz** [reply]
那就需要根据自己数据集进行微调了
